src/aks-preview/HISTORY.rst

```diff
@@ -16,6 +16,7 @@ Pending
 5.0.0b4
 ++++++++
 * Add additional unit test cases for mutable fips flags in agentpool update.
+* Add `az operation list` command to list all operations of a specific managed cluster.
```

Comment:
don't append your change note to a version that has been released, choose a new version or move yours to section pending (lines above)